### PR TITLE
Prohibit MPEG-DASH to set HTTP as application protocol.

### PR DESCRIPTION
**Pull request by utoni**
Signed-off-by: lns <matzeton@googlemail.com>

### CODE REVIEW

File: tests/result/mpeg-dash.pcap.out [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-	3	TCP 192.168.2.105:59146 -> 54.161.101.85:80 [proto: 291.265/MpegDash.AmazonAWS][ClearText][Confidence: DPI][cat: Cloud/13][1 pkts/257 bytes -> 0 pkts/0 bytes][Goodput ratio: 74/0][< 1 sec][Hostname/SNI: livesim.dashif.org][User-Agent: VLC/3.0.16 LibVLC/3.0.16][Risk: ** Known Proto on Non Std Port **][Risk Score: 50][PLAIN TEXT (GET /livesim/sts)][Plen Bins: 0,0,0,0,0,100,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
+	1	TCP 10.84.1.81:60926 <-> 166.248.152.10:80 [proto: 291/MpegDash][ClearText][Confidence: DPI][cat: Media/1][2 pkts/456 bytes <-> 2 pkts/1520 bytes][Goodput ratio: 72/92][0.30 sec][Hostname/SNI: gdl.news-cdn.site][User-Agent: Mozilla/5.0 (Linux; Android 11; SM-A715F Build/RP1A.200720.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/89.0.4389.105 Mobile Safari/537.36][Risk: ** Known Proto on Non Std Port **** Suspicious DGA Domain name **][Risk Score: 150][PLAIN TEXT (GET /as/bigo)][Plen Bins: 0,0,0,0,0,0,0,0,0,0,50,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,50,0,0,0,0]
```
Comment (IvanNardi) [inline tests/result/mpeg-dash.pcap.out]: Is that correct? Being a sub-protocol, I do expect something like HTTP/MpegDash. Am I wrong?

Comment (utoni) [reply tests/result/mpeg-dash.pcap.out]: Fixed.

**Comment by IvanNardi** [comment]
@utoni, could you rebase, please?

**Comment by IvanNardi** [comment]
> @utoni, could you rebase, please?

Thanks!

**Review by IvanNardi — APPROVED** [review]

**Pull request merged**